robotoff/models.py

```diff
@@ -74,7 +76,12 @@ class ProductInsight(BaseModel):
     # 1 = Validated
     annotation = peewee.IntegerField(null=True, index=True)
 
-    # If the insight was annotated manually, this field stores the username of the annotator.
+    # The number of votes for this annotation.
+    # Stored here for quick sorting.
+    n_votes = peewee.IntegerField(null=False, index=True)
+
```

Comment:
(sorry I comment here to have a new thread).

We add a column, but how do we handle database migration ?

Integration tests are failing in my case because my database already exists and does not have this column. Did I missed some step ?

Maybe we should use http://docs.peewee-orm.com/en/latest/peewee/playhouse.html#schema-migrations

Reply:
@zond **this is a blocker**

Reply:
As I said in the chat - I think we could migrate manually this time. AFAIU this is the first ever DB change in Robotoff, and if migrations only happen every 3 years maybe we don't need to build a migration system this time?

An alternative is to build migration support in a separate PR, and merge that one into this, and add a proper migration.

WDYT?

Reply:
IIRC we agreed to do this migration manually? Closing this discussion.